src/components/Header/Header.module.css

```diff
@@ -1,20 +1,137 @@
 .header {
-    background-color: #f8f9fa;
-    padding: 10px 20px;
-    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+  width: 100%;
+  background-color: var(--dark-theme-background);
+  padding-left: 10%;
+  padding-right: 10%;
+  padding-top: 20px;
+  padding-bottom: 20px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+  color: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 10;
+}
+.logo {
+  width: 100px;
+}
+.header-nav {
+  width: 1000px;
+  max-width: 750px;
+  margin-right: 60px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+.nav-list {
+  display: flex;
+  align-items: center;
+}
+
+.nav-list-item{
+  list-style: none;
+  color: var(--primary-white);
+  padding-right: 30px;
+}
```

Comment:
To get the last list item (the contact link) to align on the right edge, add 
```
.nav-list-item:last-child {
  margin-left: auto;
}
```
and as long as the ul fills the width here, it will line up perfectly